.github/workflows/build_rocks.yaml

```diff
@@ -29,10 +29,17 @@ on:
         type: string
         description: The cache action can either be "save" or "restore".
         default: restore
+      multiarch-awareness:
+        type: boolean
+        description: Maintain the architecture labels on the container names
+        default: false
```

Comment:
Forces the build to create a separate build for each architecture specified in the `rockcraft.yaml` platforms